Content.Server/Chemistry/ReagentEffects/RemoveReplacementAccent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Server.Speech.Components;
+
+using Content.Shared.Chemistry.Reagent;
+
+namespace Content.Server.Chemistry.ReagentEffects;
```

Comment:
newlines + order yucky

Reply:
If you don't mind me asking, what exactly about the order is bad? I checked other chemistry scripts and they use the same order. Regardless, the newlines have been fixed.